tasks/config/licenses.js

```diff
@@ -32,6 +32,7 @@ module.exports = function (grunt) {
         'UNLICENSE'
       ],
       overrides: {
+        'abbrev@1.0.7': ['MIT'],
```

Comment:
I'm actually not even sure if this is the right way to handle this... I tried to figure out the root cause of the problem (since this is only happening to me apparently), but gave up and figured it's better to just sidestep it and move forward. 😠  Should I go back and work this bug over instead?